lib/lz4frame.h

```diff
@@ -49,14 +49,30 @@ extern "C" {
 **************************************/
 #include <stddef.h>   /* size_t */
 
+/*-***************************************************************
+*  Export parameters
+*****************************************************************/
+/*!
+*  LZ4_DLL_EXPORT :
+*  Enable exporting of functions when building a Windows DLL
+*/
+#if defined(_WIN32)
+#  if defined(LZ4_DLL_EXPORT) && (LZ4_DLL_EXPORT==1)
+#    define LZ4LIB_API __declspec(dllexport)
```

Comment:
Since the macro `lz4.h` and `lz4frame.h` macros have same name,
there is a risk of redefinition warning in case both are included in same project.
It feels safer to have 2 different names,
for example `LZ4FLIB_API` for this file